### ISSUE TITLE
Style Guide: Fix table structure

### ISSUE THREAD

**Issue by lhirata** [cockroachdb/docs#3246]
(Optional, lower priority) To Do:
- Fix table structure (https://github.com/cockroachdb/docs/blob/master/STYLE.md#tables)

**Comment by Shiwanee** [comment]
Hi Ihirata,
I would like to fix the table structure. Could you please provide details?
It would be my first open-source task.
Thanks,
Shiwanee

**Comment by lhirata** [comment]
Hi @Shiwanee! Thanks for helping out!

Before you get started, check out our [Contributing Guide](https://github.com/cockroachdb/docs/blob/master/CONTRIBUTING.md). It'll help you get started with forking our repo.

For this task, we would like to get rid of the outside pipes (`|`) on our Markdown tables. If you look at a table on a doc like [ADD COLUMN](https://github.com/cockroachdb/docs/blob/master/v2.1/add-column.md), you can see that the Parameters table looks like this:

```
| Parameter | Description |
|-----------|-------------|
| `foo`        | foo foo         |
```

According to our new [Style Guide](https://github.com/cockroachdb/docs/blob/master/STYLE.md#tables), all tables should look more like this:

```
Parameter | Description 
---|---
`foo` | foo foo
```

All tables in the v2.1 directory need to be updated to this new style. To find the tables, you can probably CTRL+F `|---`.

Let me know if you have any questions! Thanks for taking this on!

**Comment by Shiwanee** [comment]
Hi Hirita!
Thanks for the inputs.
I'll go through the details that you have provided and get back to you ASAP.

Thanks,
Shiwanee

**Comment by Shiwanee** [comment]
Hi @lhirata !
I went through the Contributing Guide and came up with the HTML file for testing(add-column.md).
The formatting in the output was messed up. While studying the cause I observed that if I remove the Synopsis content, the output file renders fine.
I have attached some files for your reference. 
Could you please give me some pointers to what might be going wrong? or how can I fix this?
Thanks,
Shiwanee




[messed_up_formatting.zip](https://github.com/cockroachdb/docs/files/2104659/messed_up_formatting.zip)
[synopsis_commented_out.zip](https://github.com/cockroachdb/docs/files/2104660/synopsis_commented_out.zip)

**Comment by Shiwanee** [comment]
Hi @lhirata ,
I generated output with the changed files. When I ran the "make test" command, it gave me some broken links. I have attached the error log here.
[htmltest_errors.txt](https://github.com/cockroachdb/docs/files/2135319/htmltest_errors.txt)
Out of this some of the links are actually working, for example:
build-a-rust-app-with-cockroachdb.html 
while, some are really broken, for example, http://kubernetes.io/docs/getting-started-guides/gce/

What do you suggest next?
Thanks,
Shiwanee

**Comment by lhirata** [comment]
Hmm, it looks like the second links listed in each pairing are the ones that are broken. However, it seems like not all of the listed links are actually broken. 

Can you update the links for Kubernetes in AWS / GCE? The links look like they are in "included" content, so you will only have to update the link in one spot for each. Go to `_includes > orchestration > start-kubernetes.md`. Replace the broken links with the following:

- https://kubernetes.io/docs/setup/turnkey/gce/
- https://kubernetes.io/docs/setup/turnkey/aws/

That should help with most of the broken links. Once this is done, can you open a PR with your changes and we can work on the remaining failing tests from there? It's a little easier to see what's going on from a pull request vs. a text error log.

Thanks!

**Comment by Shiwanee** [comment]
Hi @lhirata ,
I have created the pull request, except I could not assign it to jseldess.
Please let me know if anything is pending from my side or needs change.
Thanks,
Shiwanee
